docs/topics/feed-exports.rst

```diff
@@ -275,6 +275,38 @@ as a fallback value if that key is not provided for a specific feed definition.
 * ``indent``: falls back to :setting:`FEED_EXPORT_INDENT`
 * ``store_empty``: falls back to :setting:`FEED_STORE_EMPTY`
 
+item_classes
+------------
```

Comment:
I section at the same level as `FEEDS` is not a good way to add this.

I would suggest extending the list entry with this information. See https://github.com/scrapy/scrapy/pull/4512/files#diff-b08e74952c08d71c2ff7fd960c551b10 to get an idea of how you can include details in the list itself.

I would avoid such a long code example, though. I’m not sure we need a code example at all here. A statement indicating allowed values, like the one you have above the code, should be information enough.